### PR TITLE
Fix failed specs

### PR DESCRIPTION
**Pull request by routetheta**
To pass the tests on CI, add `headless` option as default.

### CODE REVIEW

File: Gemfile.local [hunk 1]
```diff
@@ -4,6 +4,7 @@ group :test do
   gem 'factory_bot_rails'
   gem 'launchy'
   gem 'database_cleaner'
+  dependencies.reject! { |i| %w[nokogiri].include? i.name } # Ensure Nokogiri have new version
```
Comment (routetheta) [inline Gemfile.local:7]: Lower version `nokogiri` (1.11.x) causes an exception when running migration.

Comment (nishidayuya) [inline Gemfile.local:7]: more simply code is here:

```suggestion
  dependencies.reject! { |i| i.name == 'nokogiri' } # Ensure Nokogiri have new version
```

**Review by nishidayuya — APPROVED** [review]
LGTM! :tada:

I wrote one comment. It is so little change, so I approve this pull-request.

**Review by mk2 — APPROVED** [review]
LGTM!

**Review by kumojima — APPROVED** [review]

**Pull request merged**